### PR TITLE
[Cleanup] Small code clean up in HTTP1Server.

### PR DESCRIPTION
**Pull request by anayini**
Small code clean up in HTTP1Server.

### Motivation:
Cleaner code.

### Modifications:
Linting fix ups.

### Result:
No behavioral changes.  Just code clean up.

### CODE REVIEW

File: Sources/NIOHTTP1Server/main.swift [hunk 1]
```diff
@@ -147,7 +147,7 @@ private final class HTTPHandler: ChannelInboundHandler {
                 self.buffer.write(string: "line \(self.continuousCount)\n")
                 ctx.writeAndFlush(self.wrapOutboundOut(.body(.byteBuffer(self.buffer)))).map {
                     _ = ctx.eventLoop.scheduleTask(in: .milliseconds(400), doNext)
-                }.whenFailure { error in
+                }.whenFailure { _ in
```
Comment (Lukasa) [inline Sources/NIOHTTP1Server/main.swift]: Our preferred policy when we use the `_` is to provide the type annotation (e.g. `_: Error`). This helps the compiler spot (and give useful feedback) if type signatures change underneath you. 

Mind updating to do that?

**Comment by anayini** [comment]
@Lukasa - Updated!

**Comment by tomerd** [comment]
@anayini please rebase

**Comment by tomerd** [comment]
@swift-nio-bot test this please

**Comment by Lukasa** [comment]
@swift-nio-bot test this please

**Pull request merged**